### ISSUE TITLE
"Open with" ... for Threat Dragon (passing file name on the command line)

### ISSUE THREAD

**Issue by exoosh** [OWASP/threat-dragon#1313]
**Describe what problem your feature request solves**:

If Threat Dragon would accept the file name of a TM JSON file, the built-in "Open with" functionality of the desktop environment could be used to open them in Threat Dragon.

Currently you can indeed use <kbd>Open with ...</kbd> but Threat Dragon will simply ignore the passed name.

**Describe the solution you'd like**:

I'd like Threat Dragon to accept a path to a `.json` file with a threat model digestible (any format supported should be acceptable) by Threat Dragon and have it open the threat model upon opening.

Of course only a single file should open at any given point, so I suggest that it should be the first passed path (until at some point TD gains some tabbed interface or so ...).

**Comment by jgadsden** [comment]
thanks @exoosh , good to have this issue raised
I think having the CLI in place may help with this, #372
